### ISSUE TITLE
Android: Incomplete display

### ISSUE THREAD

**Issue by Feng-Xiaohan** [koreader/koreader#10388]
* KOReader version:Android
* Device:Xiaomi Mi 11Ultra

#### Issue
On the screen, the last line of the KoReader menu bar and keyboard is not displayed (Figure 1 and Figure 2).
![IMG_20230504_135216](https://user-images.githubusercontent.com/63276191/236122758-9b5a9f9a-741d-4a97-9dd1-9e0f5233348b.jpg)
![IMG_20230504_135242](https://user-images.githubusercontent.com/63276191/236122781-3859f4f9-aaab-4f24-aba6-7035f742abac.jpg)

The normal effect should be Figure 3.
![Uploading IMG_20230504_135303.jpg…]()

Tried uninstalling Koreader and then reinstalling it, but it didn't fix the problem.
Is this a problem with my settings? Is there any solution?

**Comment by poire-z** [comment]
You didn't tell your Android version, but if Android 13, see https://github.com/koreader/koreader/issues/9446#issuecomment-1297542495

**Comment by Feng-Xiaohan** [comment]
> You didn't tell your Android version, but if Android 13, see [#9446 (comment)](https://github.com/koreader/koreader/issues/9446#issuecomment-1297542495)

Sorry I misunderstood the meaning of "Android version", yes, Android 13.
Just now, I solved the problem by creating the /storage/emulated/0/koreader/patches/10-screen-offset.lua file in the connection you gave, thank you for your reply.
And sorry I filed a duplicate issue, I didn't realize that #9446 refers to the same issue.
Finally, thanks again for your help!

![Screenshot_2023-05-04-19-32-00-851_org koreader launcher](https://user-images.githubusercontent.com/63276191/236193547-1fa3816d-b996-4108-9a06-6c8c23b39686.jpg)
